build/common.ps1

```diff
@@ -158,7 +158,7 @@ Function Install-DotnetCLI {
     )
     $MSBuildExe = Get-MSBuildExe
 
-    $CmdOutLines = ((& $msbuildExe $NuGetClientRoot\build\config.props /v:m /nologo /t:GetCliBranchForTesting) | Out-String).Trim()
+    $CmdOutLines = ((& $msbuildExe $NuGetClientRoot\build\config.props /restore:false "/ConsoleLoggerParameters:Verbosity=Minimal;NoSummary;ForceNoAlign" /nologo /target:GetCliBranchForTesting) | Out-String).Trim()
```

Comment:
This is because the `Directory.Build.rsp` has an implicit restore so that you can build from the command-line with `MSBuild.exe` and not have to remember to specify it just like when you run `dotnet build`.  But in this case the script is just trying to read the variable so a restore shouldn't be run.  I could also use `/NoAutoResponse` if that's more clear

https://github.com/NuGet/NuGet.Client/pull/4880/files#diff-d316d10ab57ab324e0391b65b4c057ad2941a3e5e8cf9f92a6fd35f6611a734eR4

Reply:
> This is because the Directory.Build.rsp has an implicit restore

Oh I see it now. 
Personally I'm not a big fan of that as I tend to almost never run restore (our packages don't change) on the commandline and I always run just build. I often run `dotnet msbuild /t:build` to avoid the implicitness. This changes that, correct?

Reply:
That is correct.  But restore should be no-op and quick 😄

Reply:
I know it's quick, but it runs a complete evaluation, so while the core restore is fast, it still does take a few seconds. 
For contrast, VS' no-op is 20-30ms.